index.html

```diff
@@ -7,7 +7,10 @@ <h1 class="page-heading">EIPs
   <a href="https://discord.io/EthCatHerders"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord channel for ECH eip-editer"></a>
   <a href="https://discord.gg/EVTQ9crVgQ"><img src="https://dcbadge.vercel.app/api/server/EVTQ9crVgQ?style=flat" alt="Discord channel for Eth R&D eip-editing"></a>
   <a href="https://discord.gg/mRzPXmmYEA"><img src="https://dcbadge.vercel.app/api/server/mRzPXmmYEA?style=flat" alt="Discord server for discussions about proposals that impact Ethereum wallets"></a>
+  <a href="rss/all.xml"><img src="https://img.shields.io/badge/rss-Everything-red.svg" alt="RSS"></a>
   <a href="rss/last-call.xml"><img src="https://img.shields.io/badge/rss-Last Calls-red.svg" alt="RSS"></a>
+  <a href="rss/nonerc.xml"><img src="https://img.shields.io/badge/rss-All except ERC-red.svg" alt="RSS"></a>
+  <a href="https://eepurl.com/ikqNIP"><img src="https://img.shields.io/badge/-email%20alerts-red.svg" alt="RSS"></a>
```

Comment:
I'm hesitant to add a link to a third-party service; especially one that handles emails. Is this just a free service mailchimp provides for RSS feeds?

```suggestion
```

Reply:
Yes, it is a free service for RSS feeds. If we get past 1,000 or 5,000 people then then I'll need to make a grant or something to fund it or I'll go build something we can host ourself. Or I'll just pay it myself.

Reply:
Personally I use RSS all day. But probably the rest of the world isn't like me.

Reply:
@poojaranjan is going to see what service we already use for the Ethereum blog, and we'll go from there.

Reply:
We've just (read: literally for the first time today) started using SendPulse. Not sure I can report much so far beyond "it worked!". It's not integrated with RSS, and not sure it can be (if so, we might copy what gets done here for the blog!). 

Worth noting that MailChimp has historically been hostile to crypto projects and has shut down accounts without notice.

Reply:
Here is a way to look at this point:

1. Emails are good and will probably 10x increase the number of people that are seeing updates to EIPs (source: 2 billion people use Gmail, zero people use Google Reader).
2. Right now we don't see a great email provider we wholeheartedly recommend.
3. The people visiting our website do or do not share our values and will probably recognize a Mailchimp signup form. They are able to make their own informed decision about proceeding.
4. Merging this PR today helps some people, and going forward we can still switch to a new email host.

Reply:
Unfortunately cryptocurrencies are listed on mailchimp's [prohibited content](https://mailchimp.com/en-ca/legal/acceptable_use/) list as a "may not allow", whatever that means.

Reply:
Right, so not disallowed. Also reading that full sentence:

> Cryptocurrencies, virtual currencies, and any digital assets related to an Initial Coin Offering

depending on how you read it may not even apply since there is no offering here.

And reading in the context immediately above that of "things that cause spam complaints" very likely does not apply to EIPs.

We are good to proceed here at a minimum for the short term.

Reply:
IANAL, but when there's ambiguity, I believe it has to be read in the most permissible sense, so we'd be fine.